src/System.Memory/src/System/Buffers/Text/Utf8Formatter/Utf8Formatter.Integer.Signed.D.cs

```diff
@@ -45,9 +45,9 @@ private static bool TryFormatInt64D(long value, byte precision, Span<byte> buffe
                     }
 
                     Unsafe.Add(ref utf8Bytes, idx++) = (byte)'9';
-                    idx += FormattingHelpers.WriteDigits(223372036854775808L, 18, ref utf8Bytes, idx);
+                    FormattingHelpers.WriteDigits(223372036854775808L, 18, ref utf8Bytes, idx);
 
-                    bytesWritten = idx;
+                    bytesWritten = idx + 18;
```

Comment:
This whole special case for long.MinValue is unnecessary and can be deleted.

Reply:
Ok, I see you are not really touching this for now.

Reply:
> Ok, I see you are not really touching this for now.

Yes. Will address this along with other clean up later.